docker/lookaside_test.go

```diff
@@ -262,10 +271,16 @@ func TestRegistryConfigurationSignaureTopLevel(t *testing.T) {
 		},
 	}
 	dr := dockerRefFromString(t, "//thisisnotmatched")
+	expectDefault := "file://"
```

Comment:
Moving the default directories from `signatureTopLevel` to `configuredSignatureStorageBase` would also allow not changing these tests.